sympy/functions/special/zeta_functions.py

```diff
@@ -517,3 +517,69 @@ def eval(cls, s):
 
     def _eval_rewrite_as_zeta(self, s):
         return (1 - 2**(1 - s)) * zeta(s)
+
+
+class stieltjes(Function):
+    """
```

Comment:
I tried starting the docstring with `r`, but when I remove `\`, I get a `literal` gamma rather than a `symbolic` one.